src/frontend/screens/Game/GamePage/components/MainButton.tsx

```diff
@@ -128,6 +142,13 @@ const MainButton = ({ gameInfo, handlePlay, handleInstall }: Props) => {
 
   const is_installed = gameInfo.is_installed
 
+  const extraClassName = other.className
+
+  if (extraClassName) {
+    // do not override internal classNames
+    delete other.className
+  }
```

Comment:
I understand this `className` prop is only used to add the `mainBtn` class, right? I'd rather just hardcode the `mainBtn` class inside this component since we are not going to use that for anything else

it will be added in the old design too, but it will do nothing there if there's no CSS anyway

Reply:
Fixed